Pcap++/header/PcapFilter.h

```diff
@@ -377,31 +424,44 @@ namespace pcpp
 		 * @param[in] netmask The mask to use. The mask should match the IP versrion and be in a valid format.
 		 * Valid formats:
 		 *	 IPv4 - (X.X.X.X) - 'X' - a number in the range of 0 and 255 (inclusive)):
```

Comment:
Maybe not directly related to this PR, but there is an extra tab character just before the word "IPv4"

Reply:
Forgot to tag this, but it is also fixed in 61ac4b82